confluent_kafka/avro/serializer/message_serializer.py

```diff
@@ -213,4 +237,5 @@ def decode_message(self, message):
             if magic != MAGIC_BYTE:
                 raise SerializerError("message does not start with magic byte")
             decoder_func = self._get_decoder_func(schema_id, payload)
-            return decoder_func(payload)
+            decoded_message = _wrap(decoder_func(payload), self.registry_client.get_by_id(schema_id))
```

Comment:
What are the performance implications of this change?
It would be good with comparative numbers between master and this fix.

Also, there is no need for the temporary decoded_message variable, should `return _wrap ..`

Reply:
I did not do any performance testing, but my guess will be avro decoding will be the slowest part. On the other hand, in really many situations the message itself without the schema (or, at least, the full name of the schema which is usually the "message class") becomes close to useless.